test/test_type_info.py

```diff
@@ -36,7 +36,7 @@ def test_iinfo(self):
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_finfo(self):
         initial_default_type = torch.get_default_dtype()
-        for dtype in [torch.float32, torch.float64]:
+        for dtype in [torch.float16, torch.float32, torch.float64, torch.complex64, torch.complex128]:
```

Comment:
Can this also test the dtype attribute?

Reply:
Also, this update is great, but what about a special test case for bfloat16 that verifies it works as expected? It can't be compared with NumPy, but the test can just assert its values are what they should be.

Reply:
Addressed! The test of dtype attribute and the case for bfloat16 are both been added.

Reply:
Cool.